contrib/upstart/cilium-net-daemon.conf

```diff
@@ -1,2 +1,2 @@
 env PATH=/usr/local/clang/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin
-exec cilium -D daemon run -d eth1
+exec cilium -D daemon run --ipv4 --lb
```

Comment:
No, right now is false by default.

```golang
cli.BoolFlag{
        Destination: &config.LBMode,
        Name:        "lb",
        Usage:       "Enables load balancer mode where load balancer bpf program is attached to the interface ",
},
```

Should I change it to `cli.BoolTFlag` and remove the `--lb` flag in the `contrib/upstart/cilium-net-daemon.conf`?